include/snmp_pp/config_snmp_pp.h

```diff
@@ -55,54 +55,110 @@
 #endif
 #endif
 
-// define SNMP_PP_IPv6 if you want to use IPv6
-// #define SNMP_PP_IPv6
+/*
+ * some permanent parts from autoconf process
+ */
+#if 0
```

Comment:
I changed this line from 1 to 0 because something with my openssl installation was wrong so I disabled snmpV3. I don’t need snmpv3 and also this library supports only v1 and v2c anyway.